### PR TITLE
[FLYD-57] Cleanup code

### PR DESCRIPTION
**Pull request by bethanyhaubert**
Cleaned up code, added rubocop configs.

### CODE REVIEW

File: app/models/status.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class Status < ActiveRecord::Base
-    has_many :submissions
-    validates :name, :presence => true
+	has_many :submissions
+	validates :name, presence: true
```
Comment (clburns107) [inline app/models/status.rb]: why the change in this syntax? what does this syntax do?

Comment (bethanyhaubert) [reply app/models/status.rb]: the hash rocket `:foo => bar` is old syntax. no one uses it anymore

Comment (bethanyhaubert) [reply app/models/status.rb]: `foo: bar` does the same thing with fewer keystrokes
